### ISSUE TITLE
Get rid of fedot sources in framework  

### ISSUE THREAD

**Issue by J3FALL** [ITMO-NSS-team/BAMT#3]
Looks like BAMP uses FEDOT features, but it is not imported via requirements.
I suggest getting rid of [fedot](https://github.com/ITMO-NSS-team/BAMT/tree/master/fedot) directory. 
In order to import fedot correctly, just add requirements.txt in root and add this line:
`fedot==0.3.1`  

If a specific version of the framework is needed, it can be linked to a branch:
`pip install git+https://github.com/nccr-itmo/FEDOT@master` 
Here, the latest version from the master branch is imported.

**Comment by Anaxagor** [comment]
Yes, in the near future we are just going to put in order the work with Fedot.

**Comment by J3FALL** [comment]
> Yes, in the near future we are just going to put in order the work with Fedot.

Hi, @Anaxagor!
Good to hear that :)
It will improve compatibility in both projects.

**Comment by Anaxagor** [comment]
Fixed using of Fedot
